examples/cifar_demo/rise_demo_frontend_dev.ipynb

```diff
@@ -0,0 +1,259 @@
+    "\n",
+    "The Cifar dataset can be downloaded with the `download_cifar.py` script.\n",
+    "\n",
+    "```\n",
+    "$ ./download_cifar.py download_loc\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "cifar_loc = \"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load Cifar"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import cifar_utils\n",
+    "test_x, test_y = cifar_utils.filter_data(\n",
+    "    *cifar_utils.load_cifar(cifar_loc, cifar_filename=\"cifar_test.data\", norm=True))\n",
+    "no_norm_x, no_norm_y = cifar_utils.filter_data(\n",
+    "    *cifar_utils.load_cifar(cifar_loc, cifar_filename=\"cifar_test.data\", norm=False))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "cifar_utils.show_example_images(no_norm_x, no_norm_y, 2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Start Clipper"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# clipper_manager must be on your path:\n",
+    "import sys\n",
+    "import os\n",
+    "sys.path.append(os.path.abspath('../../management/'))\n",
+    "import clipper_manager as cm\n",
+    "# Change the username if necessary\n",
+    "user = \"ubuntu\"\n",
+    "# Set the path to the SSH key\n",
+    "key = \"\"\n",
+    "# Set the SSH host\n",
+    "host = \"\"\n",
+    "clipper = cm.Clipper(host, user, key)\n",
+    "\n",
+    "clipper.start()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "clipper.list_apps()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Create an application"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "app_name = \"cifar_demo\"\n",
+    "candidate_models = [\n",
+    "    {\"model_name\": \"sklearn_cifar\", \"model_version\": 1},\n",
+    "    {\"model_name\": \"tf_cifar\", \"model_version\": 1},\n",
+    "]\n",
+    "\n",
+    "clipper.register_application(\n",
```

Comment:
Hmm actually on second thought it makes the clipper_manager library much more confusing. [This StackOverflow answer](http://stackoverflow.com/a/5940228/814642) describes what I was doing before, and I think having all the functions just take kwargs is confusing. In the tutorial I'll make sure to explain what the arguments are.